src/neuroglancer_scripts/sharded_base.py

```diff
@@ -0,0 +1,465 @@
+# BSD 3-Clause License
+#
+# Copyright (c) 2017, Ignacio Tartavull, William Silversmith, and later authors. # noqa
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice, this
+#   list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of the copyright holder nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE # noqa
```

Comment:
ping @ylep I included the license and copyright notice per our [discussion](https://github.com/HumanBrainProject/neuroglancer-scripts/pull/35#discussion_r1432911986) . 

Some parts of the license run over the 79 character line limit, however. 

I have for now added `# noqa` to disable linter complaining. I wonder if you have any preference in resolving this issue (or is this a suitable solution?)